.travis.yml

```diff
@@ -25,6 +25,7 @@ matrix:
     - env: LANGUAGE=react-native  VERSION=1
     - env: LANGUAGE=ruby          VERSION=2.5
     - env: LANGUAGE=sonar         VERSION=3.2  GLIBC_VERSION=2.23-r3 SONARSCANNER_VERSION=3.2.0.1227
+    - env: LANGUAGE=sonarts       VERSION=10.7.0-alpine GLIBC_VERSION=2.23-r3 SONARSCANNER_VERSION=3.2.0.1227
```

Comment:
`VERSION` should be `10.7` and you should add `NODE_VERSION=10.7.0-alpine` and use it in `travis.py`

Reply:
Yes, you're right, I will follow what you did on php dockerfile